common-service-core/src/test/java/com/ca/mfaas/eurekaservice/client/util/UrlUtilsTest.java

```diff
@@ -7,10 +7,13 @@
  *
  * Copyright Contributors to the Zowe Project.
  */
-package com.ca.mfaas.discovery.staticdef;
+package com.ca.mfaas.eurekaservice.client.util;
```

Comment:
UrlUtilsTest package is not same with source UrlUtils(`com.ca.mfaas.utils`) class